src/sentry/sentry_apps/api/endpoints/sentry_app_authorizations.py

```diff
@@ -18,6 +19,18 @@
 logger = logging.getLogger(__name__)
 
 
+class SentryAppRefreshAuthorizationSerializer(serializers.Serializer):
+    client_id = serializers.CharField(required=True, allow_null=False)
```

Comment:
nit: `allow_null` looks to be `False` by default https://www.django-rest-framework.org/api-guide/fields/#allow_null

Reply:
looking at the docs, it looks like `required=True` is also a default but I'd prefer to keep this explicit since they both prevent setting other options on the serializer for the next dev that comes along 🤔